practice-one/src/styles/components/buttons.css

```diff
@@ -1,8 +1,12 @@
 .btn {
   border-radius: 6px;
   padding: 14px 30px;
+  font-size: var(--font-size-medium);
+  color: var(--color-text-link-special);
 }
 
-.btn-link {
-  color: var(--color-text-link-special);
+.btn-action {
```

Comment:
You should use the class `btn-primary` instead of `btn-action` for better naming conventions. You should refer to the class name in [Bootstrap](https://getbootstrap.com/docs/5.0/components/buttons/)